src/migration/1734071060646-FeedTypeColumn.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class FeedTypeColumn1734071060646 implements MigrationInterface {
+  name = 'FeedTypeColumn1734071060646'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "feed" ADD "type" text GENERATED ALWAYS AS (CASE WHEN "id" = "userId" THEN 'main' ELSE 'custom' END) STORED NOT NULL`);
+  }
```

Comment:
I think usually there is insert to `typeorm_metadata` as well, see examples for other generated column migrations

Reply:
Probably missed it, the auto generated migration is so noisy 🙈